src/expr.cpp

```diff
@@ -8233,6 +8248,8 @@ void SymbolExpr::Print(Indent &indent) const {
     indent.Done();
 }
 
+std::string SymbolExpr::Mangle() const { return symbol->type->Mangle() + symbol->name; }
```

Comment:
Do we need constant init value to be part of symbol mangling?

Reply:
Good point! Fixed.